pkg/daemon/update.go

```diff
@@ -556,6 +587,122 @@ func (dn *Daemon) updateKernelArguments(oldConfig, newConfig *mcfgv1.MachineConf
 	return exec.Command("rpm-ostree", args...).Run()
 }
 
+// MountOSContainer mounts the container and returns the mountpoint
+func (dn *Daemon) MountOSContainer(container string) (mnt, containerName string, err error) {
+
```

Comment:
empty line can be removed

Reply:
Ah yes, somehow missed this. Will fix it during review comment fixes